### ISSUE TITLE
Japanese characters in folder/file names are URL encoded 

### ISSUE THREAD

**Issue by fenggy09** [Velocidex/velociraptor#2454]
We've been using offline collector to collect KAPE artifacts as well as some specified folders in the local drive. When there is Japanese character in the file path the Japanese characters are URL encoded (e.g. "%3A") making it hard to understand the original folder/filename and causing the file path to be too long to have the zip file properly extracted.
**Is there a way to disable the URL encoding of collected folders/files?**

**Comment by scudette** [comment]
This is an excellent comment! Zip files do not actually have a standard path encoding. See this

https://stackoverflow.com/questions/13261347/correctly-decoding-zip-entry-file-names-cp437-utf-8-or

So we took the very conservative approach of encoding all files that are not in the ASCII printable range. But this is probably too aggressive. We probably should support utf8 in general.

What complicates matters is that some files names are not representable in Windows - for example we often have to encode "C:" and the : is not representable. If we did not escape that then the file will be created in the zip fine but Windows Explorer will break on it and it won't be easy to deal with such a zip file.

I think we should at least allow UTF8 characters and escape the bad ones.
